dali/python/backend_impl.cc

```diff
@@ -790,6 +790,13 @@ PYBIND11_MODULE(backend_impl, m) {
           ConvertTFRecordDefaultValue(converted_type, default_value);
         return new TFFeature(converted_type, converted_default_value);
       });
+  tfrecord_m.def("VarLenFeature",
```

Comment:
I wonder how we can document this new functionality that extends what TensorFlow provides in the `*Feature` API.

Reply:
Maybe add example and some explanation to [dataloading notebook](https://github.com/NVIDIA/DALI/blob/master/docs/examples/dataloading_tfrecord.ipynb) ?

Reply:
Good idea. I would also add it to "https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/supported_ops.html?highlight=varlenfeature#nvidia.dali.ops.TFRecordReader" description as this is probably the first place where users look for information.

Reply:
I updated both places with info about VarLenfeature partial_shape parameter